frontend/public/locales/fr/apply-adult-child.json

```diff
@@ -534,7 +534,7 @@
   },
   "living-independently": {
     "page-title": "Vivre de manière indépendante",
-    "description": "Vivre indépendamment de vos parents signifie que vous avez 16 ou 17 ans, que vous ne recevez aucune aide financière de vos parents et que votre résidence principale est différente de la leur.",
+    "description": "Vivre de manière indépendante signifie que vous avez 16 ou 17 ans, que vous ne recevez pas d'aide financière de vos parents et que votre résidence principale est différente de la leur.",
     "form-instructions": "Vivez-vous indépendamment de vos parents ?",
```

Comment:
There's a space before `?` in `"form-instructions"` key text. Please remove it.

![image](https://github.com/DTS-STN/canadian-dental-care-plan/assets/114004123/2fbb7ba6-5811-4d65-b98c-8ef394815219)